package.json

```diff
@@ -160,7 +160,7 @@
     "parse-github-url": "^1.0.2",
     "parse-gitignore": "^1.0.1",
     "path-key": "^3.1.1",
-    "prettyjson": "^1.2.1",
+    "prettyjson": "1.2.2",
```

Comment:
I pinned this to 1.2.2 because newer versions are not tested on CI upstream and they appear to have regressions